src/client.ts

```diff
@@ -612,6 +613,8 @@ export class Client extends EventEmitter {
   }
 
   private async setupConnectionMonitoring (stanzaInstance: NamedAgent) {
+    stanzaInstance.connectionTransfer = new ConnectionTransfer(this, stanzaInstance);
```

Comment:
This isn't the same as client-side or server-side pings, but it felt related enough that I thought it made sense to create this in the same function.